src/whats-new.js

```diff
@@ -0,0 +1,34 @@
+const whatsNew = [
+  {
```

Comment:
Can we put these in date order please? Additional things to add:
- 2018 Octicons releases
- Primer components releases
- the rest of primer releases for this year
- [Details on Details](https://github.com/muan/details-on-details)
- [Interaction of Color Systems](https://www.youtube.com/watch?v=AbpS3Z3xXE0&feature=youtu.be), CSS Conf Australia (it's about updating colors on GitHub)
- Design Better Co Podcast: [from design silos to design system](https://www.designbetter.co/podcast/diana-mounter)
- [Shipping system fonts to GitHub.com](http://markdotto.com/2018/02/07/github-system-fonts/)

Reply:
FYI if you add a lot of these releases it would probably make it worthwhile having filters! Also, is there a way we can automate this from the releases page? (Probably don't want every package posted though, just the main ones.)